src/vs/workbench/contrib/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -169,6 +169,9 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 				}
 			}
 		}));
+
+		// {{SQL CARBON EDIT}} Extension Recommendation on ADS Launch
```

Comment:
Minor: I see in some files we have a `// {{SQL CARBON EDIT}} - End` after the changes. Do you know if this is needed / if it helps the merge bot?